libr/debug/trace.c

```diff
@@ -119,6 +239,13 @@ R_API void r_debug_trace_list (RDebug *dbg, int mode) {
 			}
 		}
 	}
+	if (flag) {
+		RCore *core = dbg->corebind.core;
+		r_list_sort (info_list, cmpaddr);
+		r_core_visual_list (core, info_list, core->offset, core->blocksize,
```

Comment:
i wouldn't go that picky and i would use blocksize = 1 and core->offset passed as argument, so you dont need to add extra dpeendencies

Reply:
> i wouldn't go that picky and i would use blocksize = 1 and core->offset passed as argument, so you dont need to add extra dpeendencies

But you said not to use RCore in RDebug. core->offset is from RCore isn't it ?